### ISSUE TITLE
Question: Returned scores

### ISSUE THREAD

**Issue by Lucabenj** [RediSearch/RediSearch#2795]
Question: are there any special cases where, upon requesting to calculate the score with an algorithm such as tfidf or BM25, Redisearch does not calculate the score?

**Comment by ashtul** [comment]
@Lucabenj We are not aware of such an issue.
With FT.SEARCH, RediSearch calculates TFIDF by default, and the scorer can be changed using the `SCORER` keyword. You can use `WITHSCORES` to get the calculated score or even get a full explanation using `EXPLAINSCORE`.

**Comment by Lucabenj** [comment]
I understand. Even when you impose the sortby parameter in the code?

**Comment by ashtul** [comment]
You are correct, if you use sortby, we do not place a `SCORER` in the pipeline.
If you still want to get the score, you can add `WITHSCORES` to get back the result.

You can use `FT.PROFILE` ([link](https://redis.io/commands/ft.profile/)) and check the iterator tree and pipeline of result processors.
